components/button.js

```diff
@@ -69,11 +69,12 @@ const style = css`
 
 export default function Button ({ name, id, value, variant, type, disabled, href, onClick, children, size }) {
   if (type === 'submit') {
-    return <button type='submit' className={`button ${variant} ${size}`} disabled={disabled} name={name} id={id} onClick={onClick}>{children || value}<style jsx>{style}</style></button>
+    return <button type='submit' className={[`button`, variant, size].join(' ')} disabled={disabled} name={name} id={id} onClick={onClick} value='value'>{children || value}<style jsx>{style}</style></button>
```

Comment:
Out of curiosity, why did you choose to replace the string template with an array (and a call to `join`)? Seems to me that the string template is a bit more readable/intuitive here, no?

Reply:
@chuckwondo the classes were rendering as the string 'undefined' when these optional props didn't exist; the approach here is one I found that keeps these as non-string variables. Would love to know if you have an alternate suggestion.

Reply:
Ah, that makes sense. In that case, an alternative would be like so, but it's arguable as to whether this is any better than what you did:
```
`button ${variant || ''} ${size || ''}`
```